### PR TITLE
CDPD-45357: - Hue - adding Hue QueryProcessor to 7.2.16 blueprints templates.

### PR DESCRIPTION
**Pull request by Amlesh1902**

### CODE REVIEW

**Comment by lajosrodek** [comment]
Waiting for bobcat approval.

**Comment by shwethags** [comment]
Should we not remove DAS service in this case?
Please update spark3 template too

**Comment by Amlesh1902** [comment]
DAS service will be removed separately.
Also, please clarify which spark3 template?

**Comment by Amlesh1902** [comment]
ok, got it but the requirement was to add the Hue-QueryProcessor service in **cdp-data-engineering.bp** and **cdp-data-engineering-ha.bp** only. Also, in our original PR we have added it in these two blueprints only.

File: core/src/main/resources/defaults/blueprints/7.2.16/cdp-data-engineering-ha.bp
Comment (sidseth) [inline core/src/main/resources/defaults/blueprints/7.2.16/cdp-data-engineering-ha.bp:553]: What's the additional functionality that the query-processor brings in here?
- Does it need to be an HA deployment. 
- What resource requirements does it have on the node?
- How is it different from the das-EVENTPROCESSOR ?

Comment (Amlesh1902) [reply core/src/main/resources/defaults/blueprints/7.2.16/cdp-data-engineering-ha.bp:553]: - Initially the requirement was to add the Hue-QueryProcessor service in both cdp-data-engineering.bp and cdp-data-engineering-ha.bp for Datahub clusters wherever the DAS service presents. That's why we introduced it at both templates.
- It is a light-weight and optimised version to the DAS-eventprocessor. For now we kept it to get the customers familiar with both things, but in the upcoming release we will be sunsetting the entire DAS service.

**NOTE: This is not a new feature. This was introduced in 7.2.15 with complete testing and all the approvals. We just need it in 7.2.16 and all the upcoming release versions so that's why we are adding it in these two template files.**

Below is the original PR merged in 7.2.15 release:
https://github.com/hortonworks/cloudbreak/commit/33d2fcc78d47a55ceebc80552bf17bf171115500

Please check once with @lajosrodek for complete details and approve it.

**Comment by shwethags** [comment]
Go ahead with the commit. I can't approve as I am not added to reviewers

**Review by shwethags — APPROVED** [review]

**Pull request merged**
